sale_pricelist_global_rule/tests/test_pricelist_global.py

```diff
@@ -16,8 +16,12 @@ def setUpClass(cls):
         cls.Partner = cls.env["res.partner"]
         cls.SaleOrder = cls.env["sale.order"]
         cls.SaleOrderLine = cls.env["sale.order.line"]
-        cls.attr_size = cls.ProductAttribute.create({"name": "Size", "sequence": 1})
-        cls.attr_color = cls.ProductAttribute.create({"name": "Color", "sequence": 2})
+        cls.attr_size = cls.ProductAttribute.create(
+            {"name": "Attr Size", "sequence": 1}
```

Comment:
Use a real unique name:

```suggestion
            {"name": "
sale_pricelist_global_rule Size", "sequence": 1}
```